src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -262,6 +262,12 @@ if [[ -n $PAI_CODE_DIR ]]; then
   if [ $code_dir_size -gt 0 ] && [ $code_dir_size -le 200000000 ]; then
     hdfs dfs -get $PAI_CODE_DIR $code_dir \
       || { echo "Can not get code from HDFS. HDFS may be down."; exit 1; }
+  elif [ $code_dir_size -eq 0]; then
+    # pass when code_dir_size is 0
+    : 
+  elif [ $code_dir_size -lt 0 ]; then
```

Comment:
>$code_dir_size [](start = 9, length = 14)

in which case code_dir_size will be less than 0? should be a bug?

Reply:
It should be a bug if less than 0. So the point is whether to print an error log here or just pass and let other parts catch this bug.

Reply:
exit here seems ok.

---
In reply to: [268521074](https://github.com/Microsoft/pai/pull/2388#discussion_r268521074) [](ancestors = 268521074)